diff -up ./CMakeLists.txt.orig ./CMakeLists.txt
--- ./CMakeLists.txt.orig	2018-01-01 14:13:02.865582535 +0200
+++ ./CMakeLists.txt	2018-01-01 14:14:13.859908702 +0200
@@ -131,14 +131,14 @@ configure_file(cmake/BelrConfig.cmake.in
 	"${CMAKE_CURRENT_BINARY_DIR}/BelrConfig.cmake"
 	@ONLY
 )
-set(ConfigPackageLocation share/Belr/cmake)
+set(CONFIG_PACKAGE_LOCATION share/Belr/cmake CACHE PATH "CMake configuration file location")
 install(EXPORT ${EXPORT_TARGETS_NAME}Targets
 	FILE BelrTargets.cmake
-	DESTINATION ${ConfigPackageLocation}
+	DESTINATION ${CONFIG_PACKAGE_LOCATION}
 )
 install(FILES
 	"${CMAKE_CURRENT_BINARY_DIR}/BelrConfig.cmake"
-	DESTINATION ${ConfigPackageLocation}
+	DESTINATION ${CONFIG_PACKAGE_LOCATION}
 )
 
 add_subdirectory(build)
